README.md

```diff
@@ -157,6 +157,13 @@ npm run fetch-datasets
 env KUZU_PATH=path/to/database npm run serve
 ```
 
+### Check code style with ESLint
+
+```
+eslint --ext .js,.vue src --fix
```

Comment:
It would be better if we can predefine the command in `package.json` and use `npm run` instated of directly calling ESLint.